apps/heft/src/pluginFramework/logging/LoggingManager.ts

```diff
@@ -48,38 +52,41 @@ export class LoggingManager {
     }
   }
 
-  public getErrorStrings(fileErrorFormat?: FileErrorFormat): string[] {
+  public getErrorStrings(fileLocationStyle?: FileLocationStyle): string[] {
     const result: string[] = [];
 
     for (const scopedLogger of this._scopedLoggers.values()) {
       result.push(
         ...scopedLogger.errors.map(
-          (error) => `[${scopedLogger.loggerName}] ${LoggingManager.getErrorMessage(error, fileErrorFormat)}`
+          (error) =>
+            `[${scopedLogger.loggerName}] ` +
+            LoggingManager.getErrorMessage(error, { format: fileLocationStyle, isWarning: false })
         )
       );
     }
 
     return result;
   }
 
-  public getWarningStrings(fileErrorFormat?: FileErrorFormat): string[] {
+  public getWarningStrings(fileErrorFormat?: FileLocationStyle): string[] {
     const result: string[] = [];
 
     for (const scopedLogger of this._scopedLoggers.values()) {
       result.push(
         ...scopedLogger.warnings.map(
           (warning) =>
-            `[${scopedLogger.loggerName}] ${LoggingManager.getErrorMessage(warning, fileErrorFormat)}`
+            `[${scopedLogger.loggerName}] ` +
+            LoggingManager.getErrorMessage(warning, { format: fileErrorFormat, isWarning: true })
         )
       );
     }
 
     return result;
   }
 
-  public static getErrorMessage(error: Error, fileErrorFormat?: FileErrorFormat): string {
+  public static getErrorMessage(error: Error, options?: IFileErrorFormattingOptions): string {
     if (error instanceof FileError) {
-      return error.toString(fileErrorFormat);
+      return (error as FileError).getFormattedErrorMessage(options);
```

Comment:
Do you actually need this cast? Shouldn't TS already know that it's a `FileError`?

Reply:
I _thought_ it would, but it wasn't working for me.... though I think that may have been my IDE acting up. Looks like it works now.